pkg/controller/statusmanager/status_manager.go

```diff
@@ -57,6 +58,16 @@ func (status *StatusManager) Set(conditions ...configv1.ClusterOperatorStatusCon
 	}
 
 	oldStatus := co.Status.DeepCopy()
+
+	if reachedAvailableLevel {
+		if releaseVersion := os.Getenv("RELEASE_VERSION"); len(releaseVersion) > 0 {
+			co.Status.Versions = []configv1.OperandVersion{
```

Comment:
@smarterclayton is this right? We don't update the operator's version until all the other operands are updated? That's not what I'd expected.

Reply:
Ah, nevermind - this is correct. Not what I expected, but correct.